internal/witness/cmd/feeder/main.go

```diff
@@ -121,28 +122,22 @@ func main() {
 		glog.Exitf("Failed to set up log data: %v", err)
 	}
 	// Create the feeder.
-	feeder := feeder{
+	f := feeder{
 		logs: ctLogs,
 		w:    w,
 	}
-	// Now feed each log one by one.
-	tik := time.NewTicker(*interval)
-	for {
-		for id, ld := range feeder.logs {
-			wSize := feeder.latestSize(id)
-			glog.V(2).Infof("Tick: start feedOnce for %s (witness size %d)", ld.desc, wSize)
-			if err := feeder.feedOnce(ctx, id); err != nil {
-				glog.Warningf("Failed to feed for %s: %v", ld.desc, err)
-			}
-			glog.V(2).Infof("Tick: feedOnce complete for %s (witness size %d)", ld.desc, wSize)
-
-			select {
-			case <-ctx.Done():
-				return
-			case <-tik.C:
+	// Now feed each log.
+	wg := &sync.WaitGroup{}
+	for id, ld := range ctLogs {
```

Comment:
Yes, this makes a lot of sense - just pushed something that should be a lot less convoluted :).  The map is also just a list now because there is no need for it to be more than that.